platform-ui-skin/src/main/webapp/skin/less/ecms/skin/UIDocumentPreview/Style.less

```diff
@@ -135,6 +135,7 @@
 							margin-left: 45px;
 							margin-right: 15px;
 							position: relative;
+							word-break: break-all;
```

Comment:
The drawback of "word-break: break-all" is that it breaks ANY words (even small ones) at the edge of the width limit. We should use "word-wrap: break-word" instead since it tries to not break words in their middle when it can (for small words).